src/test/java/core/basesyntax/strategy/operationhandler/impl/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.strategy.operationhandler.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+import java.util.Map;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceOperationHandlerTest {
```

Comment:
(suggestion only) You can also add test in which balance is negative number. For me, it's really bad for fruit shop's owner)

Reply:
I am testing this case in FruitTransactionParseService. there's no way that transactions with negative quantity will be passed to this stage

Reply:
Sorry, I was inattentive)